src/global/Constants.h

```diff
@@ -10,7 +10,7 @@ static const size_t STXXL_MEMORY_TO_USE = 1024L * 1024L * 1024L * 2L;
 static const size_t STXXL_DISK_SIZE_INDEX_BUILDER = 1000 * 1000;
 static const size_t STXXL_DISK_SIZE_INDEX_TEST = 10;
 
-static constexpr size_t DEFAULT_MEM_FOR_QUERIES_IN_GB = 80;
+static constexpr size_t DEFAULT_MEM_FOR_QUERIES_IN_GB = 1;
```

Comment:
Why so little?

Reply:
I can go for `4` here. 
The problem is now, if this is too high, then the initial construction of the SortPerformanceEstimator will use that amount of money. So with a too high value here, bad things happen. Most notably, the end2end tests don't work properly. But I'll change it to `4` here (most people should have at least 8 Gigabytes on their machine) and explicitly scale down the memory for the end2end tests.